pgx-examples/bad_ideas/src/lib.rs

```diff
@@ -35,9 +35,12 @@ fn exec(
         )
     }
 
-    (
-        output.status.code(),
-        String::from_utf8(output.stdout).expect("stdout is not valid utf8"),
+    TableIterator::new(
+        [(
+            output.status.code(),
+            String::from_utf8(output.stdout).expect("stdout is not valid utf8"),
+        )]
+        .into_iter(),
```

Comment:
This is common enough that it really ought to be addressed by its own method, e.g. `TableIterator::once`.

Reply:
Agree, ya